packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -151,7 +151,7 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
             <div className={classNames.flexSlider}>
               <ColorSlider
                 className="is-hue"
-                ariaLabel={strings.hue}
+                ariaLabel={strings.hue || strings.hueAriaLabel}
```

Comment:
Should probably flip this
```suggestion
                ariaLabel={strings.hueAriaLabel || strings.hue}
```

Reply:
already mentioned that hueAriaLabel now has the default string, so strings.hue should be used if provided